charts/csi-vxflexos/values.yaml

```diff
@@ -262,6 +262,14 @@ node:
   sdc:
     # enabled: Enable/Disable SDC
     enabled: true
+      # sdcRepo: Enable/Disable SFTP/private repository details
+    sdcRepo:
+      enabled: true
+      # sdcRepoIp specifies the address of the SFTP/private repository to look up for SDC kernel files
+      # Format: "protoctol://IP"
+      sdcRepoIp: "sftp://10.255.255.255"
```

Comment:
why IP address is here ?

Reply:
Hi Prasannam this is a dummy value, customer will be updating with their actual values, this is a placeholder for private repository

Reply:
@shanmydell better to mention something like this `0.0.0.0` as this looks like actual IP

Reply:
changed to 0.0.0.0